lib/wallaroo/core/common/producer_consumer.pony

```diff
@@ -85,6 +85,19 @@ trait tag Runnable
     i_producer: Producer, msg_uid: MsgId, frac_ids: FractionalMessageId,
     i_seq_id: SeqId, latest_ts: U64, metrics_id: U16, worker_ingress_ts: U64)
 
+  fun ref _run[D: Any val](metric_name: String, pipeline_time_spent: U64, data: D,
+    key: Key, event_ts: U64, watermark_ts: U64, i_producer_id: RoutingId,
+    i_producer: Producer, msg_uid: MsgId, frac_ids: FractionalMessageId,
+    i_seq_id: SeqId, latest_ts: U64, metrics_id: U16, worker_ingress_ts: U64)
+  =>
+    // Fail()
```

Comment:
Why not use `Fail()` directly?

Reply:
I can add `use "wallaroo_labs/mort"`, but the 0.29.0 compiler doesn't know about `Fail()` when I do.

Reply:
Can you add this as a comment?